code/mob/living/carbon/human/procs/damage.dm

```diff
@@ -11,6 +11,11 @@
 	if (P.proj_data)  //ZeWaka: Fix for null.ks_ratio
 		damage = round((P.power*P.proj_data.ks_ratio), 1.0)
 
+		if (istype(P.proj_data, /datum/projectile/laser))
+			var/wound_num = rand(0, 4)
+			var/image/I = image(icon = 'icons/mob/human.dmi', icon_state = "laser_wound-[wound_num]", layer = MOB_EFFECT_LAYER)
+			src.UpdateOverlays(I, "laser_wound-[wound_num]")
+
```

Comment:
should probably be determined by the damage type of the projectile - i.e.: put in the the `D_ENERGY` section of the switch instead of using an istype, because typechecking will lead to... interesting cases such as drilling rigs leaving laser burns (drills and cutters are all subtypes of laser)

Reply:
Sure, I ended up still type checking for laser projectiles to prevent taser shots from doing it too, if looks okay